transcom/mymove
# INT B-20316 Actual Pro Gear Weights in PrimeV3

## [B-20315](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20315)

## Summary

Pro Gear Weights were getting added to the Prime V1 and Prime V2 APIs at the same time Prime V3 was getting created. That timing caused those fields to not make it into Prime V3. Later, the Simulator was updated to use Prime V3 in places that used Pro Gear Weights and this issue became much more noticeable. <- All of this was task assigned to our team.

[Brooklyn noticed this issue and fixed Prime V3 to accept and persist Pro Gear Weights](https://github.com/transcom/mymove/pull/12826/commits/7c5d8eaa848de5dc8c2798a75c5db533dfa7b63c). This ticket returns those stored values.

I was eager to move this to PI 24.5 so it could be fixed before Home Safe switched to V3.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Prime Simulator
2. Find a move for which you can update actual pro gear weights (Unrelated fields can prevent you. If it has 0 for the estimated weight and it's after the pickup date it won't let you edit anything.)
3. Add values for both or either pro gear weights and save.
4. Verify the values you just saved appear on the shipment summary
5. Verify the values appear when you go to re-edit the same shipment.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
